src/JsRoot/GraphData.tsx

```diff
@@ -112,11 +131,50 @@ export function generateGraphs(estimator: EstimatorResults) {
 			`graph_${name}_${page.data.name.replace(/ /g, '_')}.csv`
 		);
 	};
-	return gridPages
+
+	const groupPages = (pages: Page[]) => {
+		if (!groupQuantities) return pages;
+
+		const groupedPages: Page[] = [];
+
+		pages.forEach(page => {
+			if (isPage1d(page)) {
+				const group = groupedPages
+					.filter(isGroupedPage1d)
+					.find(
+						group =>
+							group.dataUnit === page.data.unit &&
+							group.axisDim1Unit === page.axisDim1.unit
```

Comment:
So the grouping is only by unit.
Can we create a union ("unique selection") of all the names of quantities and axis names  to be used on group plot ?

For example if we group together TLET and DLET I would expect axis description "TLET|DLET [MeV/cm]"

Reply:
ok